code/modules/client/country_flags.dm

```diff
@@ -2,20 +2,29 @@
 	. = ..()
 	if(CONFIG_GET(flag/ooc_country_flags))
 		spawn if(src)
-			src.country = ip2country(address)
+			src.country = ip2country(address, src)
```

Comment:
if origin is having its country modified in the proc then this src.country is probably not needed

Reply:
it's returning a big list so it probably will save on load